spec/haml_lint/configuration_loader_spec.rb

```diff
@@ -109,6 +109,146 @@
         expect { subject }.to raise_error HamlLint::Exceptions::ConfigurationError
       end
     end
+
+    context 'with an "inherits_from" directive' do
+      let(:config_file) do
+        [
+          "inherits_from: #{other_config}",
+          'skip_frontmatter: true',
+          'linters:',
+          '  AltText:',
+          '    enabled: true'
+        ].join("\n")
+      end
+      let(:other_config) { 'other_haml-lint.yml' }
+
+      before do
+        File.open(file_name, 'w') { |f| f.write(config_file) }
```

Comment:
I might be missing something here, but I don't see `file_name` being defined anywhere. Also, don't you need to do some cleanup after the test is done? (bare with me, I haven't written much code in this project).